spec/support/factory_girl.rb

```diff
@@ -11,41 +11,48 @@
     about "I love to travel. Travelling is great." # optional
   end
 
-  # factory :review do
-  #   food_rating 1
-  #   nighlife_rating 2
-  #   landmark_rating 3
-  #   cost_rating 4
-  #   weather_rating 5
-  #   culture_rating 3
-  #   description "the house with the place of the city of the french people"
-  #   user
-  #   city
-  # end
-
-  # factory :city do
-  #   food_rating 1
-  #   nighlife_rating 2
-  #   landmark_rating 3
-  #   cost_rating 4
-  #   weather_rating 5
-  #   culture_rating 3
-  #   description "the house with the place of the city of the french people"
-  #   user
-  #   city
-  # end
-  #
-  # factory :country do
-  #   food_rating 1
-  #   nighlife_rating 2
-  #   landmark_rating 3
-  #   cost_rating 4
-  #   weather_rating 5
-  #   culture_rating 3cre
-  #   description "the house with the place of the city of the french people"
-  #   user
-  #   city
-  # end
+  factory :review do
+    food_rating 1
+    nightlife_rating 2
+    landmark_rating 3
+    cost_rating 4
+    weather_rating 5
+    culture_rating 3
+    sequence(:description){ |n| "#{n}the house with the place of the city of the french people"} 
```

Comment:
Line is too long. [97/80]<br>Space missing to the left of {.<br>Space missing inside }.<br>Trailing whitespace detected.